azure_functions_worker/bindings/meta.py

```diff
@@ -25,11 +29,38 @@ def load_binding_registry() -> None:
     global BINDING_REGISTRY
     BINDING_REGISTRY = func.get_binding_registry()
 
-
-def get_binding(bind_name: str) -> object:
+    # The SDKs only support python 3.9+
+    if sys.version_info.minor > 8:
+        # Check if cx has imported sdk bindings library
+        try:
+            clients = importlib.util.find_spec('azure.functions.extension.base')
```

Comment:
nit: constant

Reply:
in setup we install extension-base if py ver >= 3.8, so this pkg shall be installed >=3.8 all the time. you can remove  clients = importlib.util.find_spec('azure.functions.extension.base') and directly import to use?

Reply:
Yeah makes sense, it shouldn't be needed